contributingGuides/PROPOSAL_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+## Proposal
+
+### Please re-state the problem that we are trying to solve in this issue.
+
+### What is the root cause of that problem?
+
+### What changes do you think we should make in order to solve the problem?
```

Comment:
```suggestion
### What changes do you think we should make in order to solve the problem?
<!-- DO NOT POST CODE DIFFS -->
```

Maybe having this here too will help eliminate them, so it's not just mentioned in the reminder